src/veri_main.ml

```diff
@@ -0,0 +1,85 @@
+open Core_kernel.Std
+open Cmdliner
+
+module Run = struct
+  open Result
```

Comment:
Usually, we're doing vice verse - opening `Bap.Std` and bap-specific modules on the toplevel, and, if needed, creating a special modules, with namespaces targeting specific cases, e.g., 
```
module Cmdline = struct
   open Cmdliner
```

In general, you should try to have your opens in the top of the file.